lib/kino/process/tracer.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Kino.Process.Tracer do
+  @moduledoc false
+
+  use GenServer
+
+  def start_link(_) do
```

Comment:
Since we start it by hand, we don't need an artificial argument :)

```suggestion
  def start_link() do
```